lib-all/index.ts

```diff
@@ -0,0 +1,25 @@
+import { DiagramEngine } from '@projectstorm/react-diagrams-core';
+import {
+	DefaultLabelFactory,
+	DefaultLinkFactory,
+	DefaultNodeFactory,
+	DefaultPortFactory
+} from '@projectstorm/react-diagrams-defaults';
+import { PathFindingLinkFactory } from '@projectstorm/react-diagrams-routing';
+
+export * from '@projectstorm/react-diagrams-core';
+export * from '@projectstorm/react-diagrams-defaults';
+export * from '@projectstorm/react-diagrams-routing';
+
+/**
+ * Construct an engine with the defaults installed
+ */
+export default (): DiagramEngine => {
```

Comment:
I would recommend not to use `default` exports. Better use named one, TypeScript can auto-import and find in suggestion imports.